modules/custom-status/custom-status.php

```diff
@@ -1620,7 +1620,7 @@ private function get_preview_link( $post ) {
 				);
 		}
 
-		$args['preview_id'] = $post->ID;
+		$args['preview'] = $post->ID;
```

Comment:
Let's remove the `preview` param from line [1614](https://github.com/Automattic/Edit-Flow/pull/442/files#diff-228291de10e40133681b1ff8414a9a54R1614) if we are adding it here (which imho we should)

Reply:
I made this change right now. I understand why line 1614 can be removed (it was so obvious once I checked the whole method once again).